apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/take_offer/review/TakeOfferReviewView.java

```diff
@@ -105,47 +105,47 @@ class TakeOfferReviewView extends View<StackPane, TakeOfferReviewModel, TakeOffe
 
         rowIndex++;
         Label priceDescription = new Label(Res.get("bisqEasy.takeOffer.review.price.price"));
-        priceDescription.getStyleClass().add(descriptionStyle);
+        priceDescription.getStyleClass().add(DESCRIPTION_STYLE);
         content.add(priceDescription, 0, rowIndex);
 
-        price = new MultiStyleLabelPane();
-        price.getStyleClass().add(valueStyle);
+        price = new TextFlow();
```

Comment:
Actually I don't see a reason why TextFlow is needed here. We have a rather static layout without expected linebreaks.